src/components/OutsideClickHandler.jsx

```diff
@@ -18,21 +19,18 @@ export default class OutsideClickHandler extends React.Component {
   }
 
   componentDidMount() {
-    if (document.addEventListener) {
-      // `useCapture` flag is set to true so that a `stopPropagation` in the children will
-      // not prevent all outside click handlers from firing - maja
-      document.addEventListener('click', this.onOutsideClick, true);
-    } else {
-      document.attachEvent('onclick', this.onOutsideClick);
-    }
+    // `capture` flag is set to true so that a `stopPropagation` in the children
+    // will not prevent all outside click handlers from firing - maja
+    this.clickHandle = addEventListener(
```

Comment:
However, since the fallback wasn't used in all of the places react-dates was adding event listeners, I don't think this should be a blocker for using consolidated-events in this project.